spec/converter_spec.rb

```diff
@@ -17,7 +17,10 @@
 
     cases.each do |numeral, value|
         it 'returns the proper value when passed a roman numeral string' do
-            expect(Converter.new.convert_roman_to_arabic(numeral)).to eq(value)
+
+            result = JSON.parse(Converter.new.convert_roman_to_arabic(numeral))
```

Comment:
Related to my comment above, this test shouldn't have to change to accommodate the new API endpoint.